### PR TITLE
Fix altMembers that Build is not able to resolve

### PR DESCRIPTION
**Pull request by TianqiZhang**
We found these when doing diff comparing for SDP change. It seems the best way is to fix the content directly instead of making workarounds in code.

### CODE REVIEW

File: xml/System.Messaging/Message.xml [hunk 1]
```diff
@@ -1847,8 +1847,8 @@ if (myObject is float) {
  -or-  
   
  The message queue is filtered to ignore the <see cref="P:System.Messaging.Message.LookupId" /> property.</exception>
-        <altmember cref="M:System.Messaging.MessageQueue.PeekByLookupId" />
-        <altmember cref="M:System.Messaging.MessageQueue.ReceiveByLookupId" />
+        <altmember cref="M:System.Messaging.MessageQueue.PeekByLookupId*" />
```
Comment (mairaw) [inline xml/System.Messaging/Message.xml]: This didn't work. I think it needs to look like this:

```suggestion
        <altmember cref="Overload:System.Messaging.MessageQueue.PeekByLookupId" />
```

File: xml/System.Messaging/Message.xml [hunk 2]
```diff
@@ -1847,8 +1847,8 @@ if (myObject is float) {
...
+        <altmember cref="M:System.Messaging.MessageQueue.ReceiveByLookupId*" />
```
Comment (mairaw) [inline xml/System.Messaging/Message.xml]:
```suggestion
        <altmember cref="Overload:System.Messaging.MessageQueue.ReceiveByLookupId" />
```

File: xml/System.Messaging/MessageLookupAction.xml [hunk 3]
```diff
@@ -23,8 +23,8 @@
   
  ]]></format>
     </remarks>
-    <altmember cref="M:System.Messaging.MessageQueue.ReceiveByLookupId" />
-    <altmember cref="M:System.Messaging.MessageQueue.PeekByLookupId" />
+    <altmember cref="M:System.Messaging.MessageQueue.ReceiveByLookupId*" />
```
Comment (mairaw) [inline xml/System.Messaging/MessageLookupAction.xml]:
```suggestion
    <altmember cref="Overload:System.Messaging.MessageQueue.ReceiveByLookupId" />
```

File: xml/System.Messaging/MessageLookupAction.xml [hunk 4]
```diff
@@ -23,8 +23,8 @@
...
+    <altmember cref="M:System.Messaging.MessageQueue.PeekByLookupId*" />
```
Comment (mairaw) [inline xml/System.Messaging/MessageLookupAction.xml]:
```suggestion
    <altmember cref="Overload:System.Messaging.MessageQueue.PeekByLookupId" />
```

File: xml/System.Messaging/MessageQueue.xml [hunk 5]
```diff
@@ -4752,7 +4752,7 @@ The name of the computer is not valid, possibly because the syntax is incorrect.
         <exception cref="T:System.InvalidOperationException">The message with the specified <paramref name="lookupId" /> could not be found.</exception>
         <exception cref="T:System.Messaging.MessageQueueException">An error occurred when accessing a Message Queuing method.</exception>
         <altmember cref="P:System.Messaging.Message.LookupId" />
-        <altmember cref="M:System.Messaging.MessageQueue.ReceiveByLookupId" />
+        <altmember cref="M:System.Messaging.MessageQueue.ReceiveByLookupId*" />
```
Comment (mairaw) [inline xml/System.Messaging/MessageQueue.xml]:
```suggestion
        <altmember cref="Overload:System.Messaging.MessageQueue.ReceiveByLookupId" />
```

File: xml/System.Messaging/MessageQueue.xml [hunk 6]
```diff
@@ -4816,7 +4816,7 @@ The name of the computer is not valid, possibly because the syntax is incorrect.
         <exception cref="T:System.Messaging.MessageQueueException">An error occurred when accessing a Message Queuing method.</exception>
         <exception cref="T:System.ComponentModel.InvalidEnumArgumentException">The <paramref name="action" /> parameter is not one of the <see cref="T:System.Messaging.MessageLookupAction" /> members.</exception>
         <altmember cref="P:System.Messaging.Message.LookupId" />
-        <altmember cref="M:System.Messaging.MessageQueue.ReceiveByLookupId" />
+        <altmember cref="M:System.Messaging.MessageQueue.ReceiveByLookupId*" />
```
Comment (mairaw) [inline xml/System.Messaging/MessageQueue.xml]:
```suggestion
        <altmember cref="Overload:System.Messaging.MessageQueue.ReceiveByLookupId" />
```

**Comment by TianqiZhang** [comment]
Thanks @mairaw , files updated :)

**Pull request merged**